fdbrpc/genericactors.actor.cpp

```diff
@@ -24,9 +24,26 @@
 #include "fdbrpc/simulator.h"
 #include "flow/actorcompiler.h"
 
-ACTOR Future<Void> disableConnectionFailuresAfter(double time, std::string context) {
+void enableConnectionFailures(std::string const& context) {
```

Comment:
I suppose this might be put somewhere else? As it is not an actor and not being called by any actors in generic actors.actor.cpp. `tester.actor.cpp` might be a better place.

Reply:
Actually, I suppose all these three functions should be in test related code...

Reply:
Move them to simulator.h now and removed this file.